cmd/ebrelayer/contract/Peggy.sol

```diff
@@ -0,0 +1,733 @@
+pragma solidity ^0.5.0;
+
```

Comment:
what is this for? It seems to be all the `.sol` in `testnet-contracts` pasted into one file? Why can't the other files be imported? Making updates to two files for one changes seems like it would cause issues in the future

Reply:
We'll be streamlining the smart contract deployment pipeline in the next phase of the project, but right now truffle is primary used for development/unit testing while remix.ethereum.org has been leveraged for ropsten deployment and live testing. Remix requires a slightly different file structure than truffle for contract compilation - this file is the 'remix ready' version.